contracts/wrapperContracts/WrapperBase.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity 0.4.18;
+
+
+
+import "../Withdrawable.sol";
+import "../PermissionGroups.sol";
+
+
+contract WrapperBase is Withdrawable {
+
+    PermissionGroups wrappedContract;
+
+    struct DataTracker {
+        address [] approveSignatureArray;
+        uint lastSetNonce;
+    }
+
+    DataTracker[] internal dataInstances;
```

Comment:
I get the feeling this is suppose to be mapping. but will keep reading.

Reply:
you still have the feeling?